opm/parser/eclipse/Deck/Deck.cpp

```diff
@@ -132,6 +132,14 @@ namespace Opm {
         m_dataFile("")
     {}
 
+    Deck::Deck( std::initializer_list< DeckKeyword > ilist ) :
+        Deck( std::vector< DeckKeyword >( ilist.begin(), ilist.end() ) )
+    {}
```

Comment:
... and for convenience, sources.
http://en.cppreference.com/w/cpp/utility/initializer_list
http://en.cppreference.com/w/cpp/language/value_category